### PR TITLE
Create calendars with random recipes instead of based on a passed in …

### PR DESCRIPTION
**Pull request by jstrojny**
…calendar

This also includes the start of pushing the repository layer down into the controller layer instead of middleware layer.

### CODE REVIEW

File: middleware/householdMiddleware.go [hunk 1]
```diff
@@ -140,12 +139,12 @@ func (hm HouseholdMiddleware) CreateCalendar(w http.ResponseWriter, r *http.Requ
 	if userErr != nil {
 		json.NewEncoder(w).Encode(userErr.Error())
 	} else {
-		var calendar models.Calendar
-		_ = json.NewDecoder(r.Body).Decode(&calendar)
 		bearerToken := r.Header.Get("Authorization")
 		user, _ := hm.um.repository.GetUserByAccessToken(strings.ReplaceAll(bearerToken, "Bearer ", ""))
-		calendar.HouseholdID, _ = primitive.ObjectIDFromHex(user.HouseholdId)
-		payload, err := hm.controller.CreateCalendar(calendar, hm.calendarRepo)
+		// Should this be a separate model, something like CalendarCreateRequest that only has the StartDate?
```
Comment (alexandra-strojny) [inline middleware/householdMiddleware.go:144]: in a pure REST sense, yes, but I'm cool with this personally

File: main.go [hunk 2]
```diff
@@ -71,13 +71,13 @@ func main() {
 	// to more cleanly manage it
 
 	// Build router from middleware
-	var router = router.NewTastyBoiRouter(userMiddleware, recipeMiddleware, ingredientMiddleware, serverMiddleware, householdMiddleware)
+	var tastyRouter = router.NewTastyBoiRouter(userMiddleware, recipeMiddleware, ingredientMiddleware, serverMiddleware, householdMiddleware)
```
Comment (alexandra-strojny) [inline main.go:74]: lmao wat.

**Review by alexandra-strojny — APPROVED** [review]

File: main.go (hunk 2 above)
Comment (jstrojny) [reply main.go:74]: var router actually conflicts with the package name so I renamed it not. I already used `r` below so this was the the first thing I thought of.

File: middleware/householdMiddleware.go (hunk 1 above)
Comment (jstrojny) [reply middleware/householdMiddleware.go:144]: Yea at some point I'll go through and implement CQRS but there's other cleanup projects *side-eyes tests* that I need to do first.

File: main.go (hunk 2 above)
Comment (alexandra-strojny) [reply main.go:74]: mmmmmm.... router

**Pull request merged**